bin/enable-executable

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env ruby
+
+require 'fileutils'
+require 'pathname'
+
 puts 'Ensuring test files are executable...'
 
-Dir.glob('**/*test.rb').each do |f|
+test_files = Dir.glob(File.join(__dir__, '../exercises/**/*test.rb'))
+bin_files = Dir.glob(File.join(__dir__, '*'))
+
+(test_files + bin_files).each do |f|
   if File.executable?(f)
     print '.'
   else
-    print "Adding exec bit to #{f}."
-    File.chmod(0744, f)
+    relative_path = Pathname.new(f).relative_path_from(Pathname.pwd)
+    print "\nAdding exec bit to #{relative_path}."
```

Comment:
I guess the simplest way for me to express myself is to show the progression from master, to the next smallest change for what is wanted to be accomplished (Still room for improvement in this example without adding dependencies?) for  what is hopefully identical behavior to what your changes introduce.

This is the code from master, with the intended changes that you have introduced to enable the executable bit on the bin folder.

``` ruby
#!/usr/bin/env ruby
puts 'Ensuring test files are executable...'

files = Dir.glob('**/*test.rb') + Dir.glob('bin/**')
files.each do |f|
  if File.executable?(f)
    print '.'
  else
    print "\nAdding exec bit to #{f}."
    File.chmod(0744, f)
  end
end
puts
```

This is the proposed change as it sits right now.
```ruby
#!/usr/bin/env ruby

require 'fileutils'
require 'pathname'

puts 'Ensuring test files are executable...'

test_files = Dir.glob(File.join(__dir__, '../exercises/**/*test.rb'))
bin_files = Dir.glob(File.join(__dir__, '*'))

(test_files + bin_files).each do |f|
  if File.executable?(f)
    print '.'
  else
    relative_path = Pathname.new(f).relative_path_from(Pathname.pwd)
    print "\nAdding exec bit to #{relative_path}."
    FileUtils.chmod('+x', f)
  end
end
puts
```

This is done without adding two full libraries which is adding dependencies which aren't needed, which means less lines of code, and less potential for bugs.